internal/services/datafactory/data_factory_integration_runtime_azure_ssis_resource.go

```diff
@@ -107,6 +107,12 @@ func resourceDataFactoryIntegrationRuntimeAzureSsis() *pluginsdk.Resource {
 				ValidateFunc: validation.IntBetween(1, 16),
 			},
 
+			"user_assigned_identity_credential_name": {
```

Comment:
```suggestion
			"credential_user_assigned_identity_name": {
```

Reply:
Done. I struggled with the name since it is a mouthful